contracts/Aavegotchi/WearableDiamond/libraries/LibDiamond.sol

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: MIT
```

Comment:
The file name should be changed.

Reply:
The library name?

Reply:
The file name is LIbDiamond, but the library name is WearableLibDiamond which I think is confusing?

Reply:
gotcha